components/Form.js

```diff
@@ -170,12 +166,13 @@ const InputWrapper = styled.label`
 const Input = styled.input`
   width: ${(props) => (props.default ? "100%" : "6.5rem")};
   height: 3rem;
-  font-size: ${StyledCaption};
   border-radius: 1em;
+  background-color: rgba(235, 237, 242, 1);
   border: 1px solid var(--clr-accent);
   padding: 1em 0.7em;
   &:focus {
-    border: 1.1px solid var(--clr-accent);
+    outline: none;
```

Comment:
You should never remove the outline for accessibility reasons. [See MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns)